### ISSUE TITLE
Hiding react-error-overlay errors that are shown

### ISSUE THREAD

**Issue by ofergoli** [codesandbox/codesandbox-client#5890]
# 🌈 Feature

Someone knows or able to create a sandbox that not showing this error page made by react-error-overlay ?
<img width="497" alt="Screen Shot 2021-06-11 at 9 48 19 AM" src="https://user-images.githubusercontent.com/8341557/121643531-95af4c80-ca9a-11eb-912d-3d972690f9f7.png">

Thanks (:

**Comment by DeMoorJasper** [comment]
Hey, we use react-overlay to show the errors in your application both from compilation and runtime.

What is the behaviour you expect in this case?

**Comment by ofergoli** [comment]
That it won't show the overlay errors.
When someone typing something invalid

**Comment by DeMoorJasper** [comment]
So you don't want to preview to update unless you've saved the file?

**Comment by ofergoli** [comment]
No.. I'm embedding Iframe in my website. and when someone Writes down a typo I don't want to show these alerts.
Its website for coding practice

**Comment by ofergoli** [comment]
Is there any option to setup that somehow in a sandbox?
I saw that on react NODE_ENV=production it wont show, but when setting .env with that, it crashed something internally in codesandbox i guess

**Comment by DeMoorJasper** [comment]
Alright got it, so you want the sandbox to be in production mode. We don't really support this yet and environment variables aren't supported either, there's an open feature request for this: #5861

**Comment by ofergoli** [comment]
Ok thanks a lot ☺️

**Comment by ofergoli** [comment]
Any update on that? (: @DeMoorJasper

**Comment by DeMoorJasper** [comment]
No, haven't really gotten a chance to look into this yet.

Backlog for sandpack is pretty big and we prioritise things that impact the main experience like performance, stability and some missing features like proper ESModule support
